test/sai_test/config/route_configer.py

```diff
@@ -201,6 +206,65 @@ def t0_route_config_helper(
         print(
             "Create route for server with in ip {}/{}".format(test_obj.servers[2][0].ipv4, 24))
 
+    if is_create_route_for_nhopgrp:
+        nhpv4_list, nhpv6_list = [], []
```

Comment:
can we use a loop to create all of those nhops?

Reply:
yes, already modified